Update enroll-device-linux.md (#3569)
MicrosoftDocs/memdocs: memdocs/intune/user-help/enroll-device-linux.md

   2.  An on-screen confirmation appears when your device is enrolled and ready-to-use for work. You can begin using your device for work right away. 
   3. Sign in to Microsoft Edge with your work or school account to access your org's internal websites.   

> [!NOTE]

> > Ubuntu on WSL2 is not a supported scenario and the requests is in on our backlog and we are investigating ways to add them.

Reviewer: ```suggestion
> Currently, Ubuntu on WSL2 is not supported but we are investigating ways to add support for it in the future..
```
